include/MyJobs.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "macros.h"
+
+class MyJobs {
+
+    string command;
+    pid_t pid;
+
+    static vector<MyJobs> m_jobs;
```

Comment:
Why does `MyJobs` contain a static vector of itself? This seems counter-intuitive. You probably want to define a `Job` struct, and then have some OTHER class hold a vector of those jobs